tests/ColumnSchemaTest.php

```diff
@@ -197,4 +197,17 @@ public function testDbTypeCastBit()
 
         $this->assertSame('01100100', $tableSchema->getColumn('bit_col')->dbTypecast('01100100'));
     }
+
+    public function testPrimaryKeyOfView()
```

Comment:
How about add to test  assert:
tableSchema->getPrimaryKey() == ['C_id_1', 'C_id_2';]